at_secondary/at_secondary_server/lib/src/verb/handler/proxy_lookup_verb_handler.dart

```diff
@@ -94,7 +94,7 @@ class ProxyLookupVerbHandler extends AbstractVerbHandler {
       await outBoundClient.connect(handshake: false);
     }
     // call lookup with the query. Added operation as all to get key's value and metadata for caching
-    return await outBoundClient.lookUp('all:$query', handshake: false);
+    return outBoundClient.lookUp('all:$query', handshake: false);
```

Comment:
Why no await?

Reply:
While returning we don't need awaits. It will return only the values not instances. So no need to use awaits in returns.